.github/workflows/dashwallet.yml

```diff
@@ -84,12 +84,6 @@ jobs:
         APP_DISTRIBUTION_KEY_JSON: ${{secrets.APP_DISTRIBUTION_KEY_JSON}}
         GC_STORAGE_SERVICE_ACCOUNT_KEY_JSON: ${{secrets.GC_STORAGE_SERVICE_ACCOUNT_KEY_JSON}}
 
-    - name: Extract Secrets
-      run: |
-        echo "$GOOGLE_SERVICES_JSON" > wallet/google-services.json
-      env:
-        GOOGLE_SERVICES_JSON : ${{secrets.GOOGLE_SERVICES_JSON}}
-
```

Comment:
Seems like I saw this duplicate secret extraction somewhere else, maybe on the `master` branch?

Reply:
Yes. I'll clean it up in the low-storage PR.